internal/commands/app/create_inputs.go

```diff
@@ -16,12 +16,14 @@ import (
 
 var (
 	flagDirectory      = "app-dir"
-	flagDirectoryShort = "c"
+	flagDirectoryShort = "p"
 	flagDirectoryUsage = "the directory to create your new Realm app, defaults to Realm app name"
 
-	flagDataSource      = "data-source"
-	flagDataSourceShort = "s"
-	flagDataSourceUsage = "include to link an Atlas cluster to your Realm app, defaults to first available"
+	flagCluster      = "cluster"
+	flagClusterUsage = "include to link an Atlas cluster to your Realm app"
+
+	flagDataLake      = "data-lake"
+	flagDataLakeUsage = "include to link an Atlas data lake to your Realm app"
```

Comment:
[thoght for future] I wonder if it would be nice to have datalake as an alias to the flag. I like this as is rn and it's not important, just a thought

Reply:
i don't love `--data-lake` either, so much so I went to google to see just how we actually refer to them in the wild.  the fact we seem to use "Data Lake" everywhere sadly seems to want us to keep using the hyphenated `--data-lake`

(we can use this as motivation to figure out auto-complete, so the flag just writes itself...)